tests/cases/test_raft.rs

```diff
@@ -2260,6 +2260,154 @@ fn test_non_promotable_voter_which_check_quorum() {
     assert_eq!(nt.peers[&2].leader_id, 1);
 }
 
+#[test]
+// test_disruptive_follower tests isolated follower,
```

Comment:
Please use [doc comment](https://doc.rust-lang.org/reference/comments.html#doc-comments) and put the `#[test]` attribute below the comments.